packages/server/src/db/records/like.ts

```diff
@@ -19,8 +22,10 @@ export const createTable = async (db: Kysely<PartialDB>): Promise<void> => {
   await db.schema
     .createTable(tableName)
     .addColumn('uri', 'varchar', (col) => col.primaryKey())
+    .addColumn('cid', 'varchar', (col) => col.notNull())
     .addColumn('creator', 'varchar', (col) => col.notNull())
     .addColumn('subject', 'varchar', (col) => col.notNull())
+    .addColumn('subjectCid', 'varchar', (col) => col.notNull())
```

Comment:
Not intended for this PR, but it might be time we sprinkled some column indexes in here.  E.g. we want to make it quick to query liked-by, which would be supported by an index `(subject, subjectCid)`.

Reply:
agreed 👍